scss/_patterns_side-navigation.scss

```diff
@@ -210,8 +227,43 @@
     margin-bottom: $sp-unit * 2 - $spv-nudge * 2;
 
     &::before {
+      @include vf-icon-chevron($colors--light-theme--text-inactive);
       transform: rotate(90deg);
     }
+
+    &[aria-expanded='true'] {
+      background-color: $color-transparent;
+
+      &:hover {
+        background: $colors--theme--background-hover;
+        color: $colors--theme--text-default;
+      }
+    }
+  }
+
+  // FIXME: Temporary workaround for unsupported theme colors in vf-icon-chevron
+  .p-side-navigation__toggle--in-drawer.is-light::before {
```

Comment:
Same comment as in the previous file, about merging those into single declarations.